hedera-node/docs/system-accounts-operations.md

```diff
@@ -0,0 +1,184 @@
+# System accounts
+
+## Definition
+The Hedera network reserves the first 1000 entity numbers for its own uses, and any account with a number in this reserved range is referred to as a system account.
+
+The following addresses are reserved for the Hedera precompiled contracts:
+- 0x167 (0.0.359) - HTS precompile;
+- 0x168 (0.0.360) - ExchangeRate precompile;
+- 0x169 (0.0.361) - PRNG precompile.
```

Comment:
The set of addresses here can also expand in the future.   We are already working on 0x16a for HAS system contracts